rpc/src/main/java/org/dst/rpc/core/common/constants/CodecConstants.java

```diff
@@ -0,0 +1,67 @@
+package org.dst.rpc.core.common.constants;
+
+import org.dst.rpc.core.exception.CodecException;
+
+
+public interface CodecConstants {
+
+  short MAGIC_HEAD = 9527;
```

Comment:
What does this field do?
If it indicates the magic number for the protocol, I'd like to name it `PROTOCOL_MAGIC_NUMBER`.

Reply:
In general, a protocol will have a MagicNumber at the front of it protocol header in order to perform a simple check when we transport data packages. This is a fail-fast strategy. Dubbo and Motan protocol both have the MagicNumber. When Sofa-Bolt not use a MagicNumber in its protocol, it takes a Cyclic-Redundancy-Check, an other check strategy of correctness of data packages.
It's a fail-fast strategy, means it can not guarantee that it will make a right check each time.

Reply:
got it！